src/pages/ScriptPage.tsx

```diff
@@ -520,8 +520,8 @@ class Script extends React.Component {
                 ele.remove();
               })
               .catch((error) => {
-                this.setState({ errorMessage: "An error occurred during edge deletion." });
-                console.error(`An error occurred during edge deletion: ${error}`);
+                this.setState({ errorMessage: error.response?.data ?? "Unexpected error occurred." });
```

Comment:
What is the reason to remove specific messages and replace with duplicate code ...

see also 2 other places below